ci-operator/jobs/integr8ly/cloud-resource-operator/integr8ly-cloud-resource-operator-MGDAPI-3425-GCP-presubmits.yaml

```
@@ -0,0 +1,317 @@
+presubmits:
+  integr8ly/cloud-resource-operator:
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - ^MGDAPI-3425-GCP$
+    - ^MGDAPI-3425-GCP-
+    cluster: build01
+    context: ci/prow/coverage
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci.openshift.io/generator: prowgen
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-integr8ly-cloud-resource-operator-MGDAPI-3425-GCP-coverage
+    rerun_command: /test coverage
+    spec:
+      containers:
+      - args:
+        - --gcs-upload-secret=/secrets/gcs/service-account.json
+        - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
+        - --report-credentials-file=/etc/report/credentials
+        - --secret-dir=/secrets/cloud-resource-operator-codecov-token
+        - --target=coverage
+        command:
+        - ci-operator
+        image: ci-operator:latest
+        imagePullPolicy: Always
+        name: ""
+        resources:
+          requests:
+            cpu: 10m
+        volumeMounts:
+        - mountPath: /secrets/cloud-resource-operator-codecov-token
+          name: cloud-resource-operator-codecov-token
+          readOnly: true
+        - mountPath: /secrets/gcs
+          name: gcs-credentials
+          readOnly: true
+        - mountPath: /etc/pull-secret
+          name: pull-secret
+          readOnly: true
+        - mountPath: /etc/report
+          name: result-aggregator
+          readOnly: true
+      serviceAccountName: ci-operator
+      volumes:
+      - name: cloud-resource-operator-codecov-token
+        secret:
+          secretName: cloud-resource-operator-codecov-token
+      - name: pull-secret
+        secret:
+          secretName: registry-pull-credentials
+      - name: result-aggregator
+        secret:
+          secretName: result-aggregator
+    trigger: (?m)^/test( | .* )coverage,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - ^MGDAPI-3425-GCP$
+    - ^MGDAPI-3425-GCP-
```

```suggestion
    - ^mgdapi-3425-gcp$
    - ^mgdapi-3425-gcp-
```
Kinda minor but I think using lower case branch would be better if possible :thinking: 

I think there will be issues with the `images` prow check when the final feature branch tries to merge to master, which required it to be overriden. (Happened for STS feature branch before - https://github.com/integr8ly/cloud-resource-operator/pull/315#issuecomment-1135925720 and guide also uses a lower case feature branch name - https://github.com/RHCloudServices/integreatly-help/blob/master/guides/branching-operator.md)

---

Sounds good I'll make the change

---

@KevFan Actually will I be able to do that because the branch names are uppercase `MGDAPI`

---

@carlkyrillos No, I think lowercased branch names will need to be created by the maintainers before making this change :thinking:

---

@laurafitzgerald CCing you about the branch name change. Does that sounds good to you? If so I can get started on renaming(recreating) the [GCP branch](https://github.com/integr8ly/integreatly-operator/tree/MGDAPI-3425-GCP) to be all lowercase.